MAGECLOUD-3409 update web property (#4205)
magento/devdocs: guides/v2.1/cloud/project/project-conf-files_magento-app.md

You can specify the following attributes for the `web` property:

Attribute | Description
--------- | -----------
`document_root` | The path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.

> `passthru` | The URL used in the event a static file or PHP file could not be found. This would typically be your applications front controller, often `/index.php` or `/app.php`.

Reviewer: ```suggestion
`passthru` | The URL used in the event that a static file or PHP file cannot be found. This URL is typically the front controller for your applications, often `/index.php` or `/app.php`.
```

Reviewer: Question-- In our docs, I noticed that we use URL sometimes when we mean path --    
/index.php is a path.     In this case, I think that the path specified is appended to the site URL. Maybe this could be clearer. Maybe I am splitting hairs?

> `index_files` | To use a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, your whitelist should include an element that matches the filename, like `- \.html$`.

Reviewer: ```suggestion
`index_files` | Specifies a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, make sure that your whitelist includes an element that matches the filename, like `- \.html$`.
```

`blacklist` | A list of files that should never be executed. Has no effect on static files.
`whitelist` | A list of static files (as regular expressions) that can be served. Dynamic files (for example, PHP files) are treated as static files and have their source code served, but they are not executed.

> `expires` | The number of seconds whitelisted (that is, static) content should be cached by the browser. This enables the cache-control and expires headers for static content. The `expires` directive and resulting headers are left out entirely if this is not set.

Reviewer: ```suggestion
`expires` | The number of seconds to cache whitelisted (that is, static) content in the browser. This attribute enables the cache-control and expires headers for static content. If this value is not set, the `expires` directive and resulting headers are not included when serving static content files.
```

> Contrary to standard `.htaccess` approaches that accept a _blacklist_ and allow access to everything not on a specific list, we accept a _whitelist_, which means that anything not matched will trigger a 404 error and will be passed through to your `passthru` URL.

Reviewer: ```suggestion
Contrary to standard `.htaccess` approaches that accept a _blacklist_ and allow access to everything not on a specific list, we accept a _whitelist_, which means that any request that does not match triggers a 404 error and passes through to the  URL specified by the `passthru` attribute.
```

Our default configuration allows the following:

-  From the root (`/`) path, only web, media, and `robots.txt` files can be accessed  

>     For versions 2.1.4 to 2.1.10, the rules entry for the `robots.txt` file:

Reviewer 2: Maybe it's just me, but the lead-in for this example sounds incomplete. I would expect to see a convention similar to line 105. For example:

> For versions 2.1.4 to 2.1.10, the following example shows the required rules entry for the `robots.txt` file:

Which begs the question, what about versions other than 2.1.4 to 2.1.10? Is there a different entry for other 2.1.x versions that I need to be aware of?

Author: Yeah, I can make that change.
It is the exception to the default, so maybe it can state that, too.

>     For versions 2.1.4 to 2.1.10, the rules entry for the `robots.txt` file:

Reviewer: ```suggestion
    For versions 2.1.4 to 2.1.10, the configuration includes the following rules entry for the `robots.txt` file:
```

    
    ```yaml
    /robots\.txt$:
        passthru: "/media/robots.txt"
    ```

-  From the `~/pub/static` and `~/pub/media` paths, any file can be accessed

> The following displays the default set of web accessible locations associated with an entry in [`mounts`](#mounts):

Reviewer: ```suggestion
The following example shows the default configuration for a set of web accessible locations associated with an entry in [`mounts`](#mounts):
```
